test/joke_test.rb

```diff
@@ -14,4 +15,21 @@ def test_it_has_attributes
     assert_equal "Why did the strawberry cross the road?", joke.setup
     assert_equal "Because his mother was in a jam.", joke.punchline
   end
+
+  def test_it_has_a_user
+    joke = Joke.new(1, "Why did the strawberry cross the road?", "Because his moether was in a jam.")
+    sal = User.new("Sal")
+    assert_equal "Sal", sal.name
+    assert_equal [], sal.jokes
```

Comment:
The joke test was already written for you and working, so you just had to make it pass.  You're testing things about a user here, so it should go into a new test file for a user.